pricegraph/src/api/transitive_orderbook.rs

```diff
@@ -454,4 +456,22 @@ mod tests {
             pricegraph.transitive_orderbook(Market { base: 3, quote: 2 }, None);
         assert!(transitive_orderbook.asks.is_empty() && transitive_orderbook.bids.is_empty());
     }
+
+    #[test]
+    fn transitive_orderbook_with_unlimited_order_and_large_balance() {
+        let pricegraph = Pricegraph::new(vec![Element {
+            user: Default::default(),
+            balance: U256::MAX,
+            pair: TokenPair { buy: 0, sell: 1 },
+            valid: Validity { from: 0, to: 0 },
+            price: PriceFraction {
+                numerator: u128::MAX,
+                denominator: 10u128.pow(18),
+            },
+            remaining_sell_amount: 10u128.pow(18),
+            id: 0,
+        }]);
+
+        pricegraph.transitive_orderbook(Market { base: 1, quote: 0 }, None);
```

Comment:
I assume this is the regression test. From the test name/code I cannot see what this is really testing (there seems to be no assertion). Maybe adding a `doesnt_oom` would be nice?

Reply:
I added a comment explaining the regression test and `doesnt_oom` to the title.